app/client/templates/elements/eventsTable.js

```diff
@@ -49,8 +49,9 @@ Template['elements_event_table'].helpers({
             searchQuery = TemplateVar.get('search'),
             limit = TemplateVar.get('limit'),
             collection = Events,
-            selector = this.ids ? {_id: {$in: this.ids}} : {};
-
+            selector = this.ids ? {_id: {$in: this.ids.slice(Number((limit+1)*-1))}} : {}; 
```

Comment:
I don't have time to test it myself, but please all make sure you test this properly, as you circumvent the sorting of the cursor...